wgpu-types/src/lib.rs

```diff
@@ -881,56 +1191,90 @@ pub enum SwapChainStatus {
     OutOfMemory,
 }
 
+/// Operation to perform to the output attachment at the start of a renderpass.
+///
+/// This must be clear if it is the first renderpass rendering to a swap chain image.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 #[cfg_attr(feature = "peek-poke", derive(PeekPoke))]
 pub enum LoadOp {
+    /// Clear the output attachment with the clear color.
     Clear = 0,
+    /// Do not clear output attachment. When rendering to a swap chain image, you can only use this
+    /// when a previous renderpass has cleared the swap chain image.
     Load = 1,
 }
 
+/// Operation to perform to the output attachment at the end of a renderpass.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 #[cfg_attr(feature = "peek-poke", derive(PeekPoke))]
 pub enum StoreOp {
+    /// Clear the render target
```

Comment:
let's say that this should be used when you don't care about the contents, and it can be faster